content/articles/creating-serverless-functions-faas-with-aws-lambda-and-golang/index.md

```diff
@@ -0,0 +1,292 @@
+Creating Serverless Functions (Faas) with AWS Lambda and Golang
+
+Serverless architecture is a new paradigm that is being used in the development of modern applications. The term "serverless" refers to the abstraction of "servers" from application development. Function-as-a-Service (FaaS) is a serverless method of running modular code on the cloud. You can use FaaS to write code that is triggered by events. For example, when a user clicks on a button in your web application, a specific cloud function is executed.
+
+We will not only look at how to use FaaS in this tutorial, but we will also construct some for others to use. We'll be working in Golang, but the same method can be applied to Python, Node, Java, or any other language or framework you want.
+
+### Requirements
+
+* [AWS](https://aws.amazon.com) account
+* Go 1.x installed on your machine
```

Comment:
Be more specific.

I believe you are talking about AWS instance.

Reply:
I was talking about creating an AWS account. I'll fix it. Thank you.